### ISSUE TITLE
爱可可微博热门分享(4.6)

### ISSUE THREAD

**Issue by fly51fly** [fly51fly/aicoco#120]
No 1. <a href="https://weibo.com/1402400261/O8q7fqGsi">RAG进化之路</a>
No 2. <a href="https://weibo.com/1402400261/O8ivvFIdX">'Bili2text - Bilibili视频转文字，一步到位，输入链接即可使用' GitHub: github.com/lanbinshijie/bili2text ......</a>
No 3. <a href="https://weibo.com/1402400261/O8k3kbPso">'GPTS-Prompt-Collection - 收集GPTS的prompt / Collect the prompt ......</a>
No 4. <a href="https://weibo.com/1402400261/O8tAX7jU3">从0到1构建一个MiniLLM</a>
No 5. <a href="https://weibo.com/1402400261/O8u2bg0zt">'MaxKB - 基于 LLM 大语言模型的知识库问答系统，开箱即用，支持快速嵌入到第三方业务系统' GitHub: ......</a>
No 6. <a href="https://weibo.com/1402400261/O8pDdpdb8">#免费##抽奖# 携手@异步图书  送出3本《大语言模型：基础与前沿》</a>
No 7. <a href="https://weibo.com/1402400261/O8t6xlOOj">ai-researcher：AI 研究助手，基于 Anthropic Claude 和 SERPAPI，可以对给定的主题进行详细的研究。它将研究过程分解为子主题，为每个子主题生成单独的报告，然后将它们合并为最终的综合报告</a>
No 8. <a href="https://weibo.com/1402400261/O8skhukz7">'StyleLLM 文风大模型 - StyleLLM文风大模型：基于大语言模型的文本风格迁移项目，提供了四个基于中国四大名著训练的模型。Text ......</a>
No 9. <a href="https://weibo.com/1402400261/O8hyyppLO">生成式AI高峰已过？</a>
No 10. <a href="https://weibo.com/1402400261/O8j7p9nAF">Dot：使用大型语言模型(LLM)和检索增强生成(RAG)进行文档交互的独立应用，面向本地使用大型语言模型（LLM）和检索增强生成（RAG）进行文档交互的独立应用程序</a>
No 11. <a href="https://weibo.com/1402400261/O8ujEAJtY">AI生成内容泛滥冲击Google Books</a>
No 12. <a href="https://weibo.com/1402400261/O8t3nmIs9">FreeAskInternet：完全免费、私有且本地运行的搜索聚合器 & 答案生成器，无需 GPU(使用免费的ChatGPT3.5 API)，可用Docker运行</a>
No 13. <a href="https://weibo.com/1402400261/O8myh1Zvi">晚安～ [月亮] ​</a>
No 14. <a href="https://weibo.com/1402400261/O87JkzOO5">代号Lavender的AI“暗杀”系统：以色列AI军事应用引发伦理和道义拷问</a>
No 15. <a href="https://weibo.com/1402400261/O8sSegIG5">mamba-tiny：简单、紧凑的 Mamba SSM（State Space Model）在 PyTorch 中的实现</a>
No 16. <a href="https://weibo.com/1402400261/O8tEkBkex">Rule-based Retrieval：一个 Python 软件包，用于创建和管理检索增强生成 (RAG) 应用，支持高级过滤功能，集成了 OpenAI 和 Pinecone，用于文本生成和高效的向量数据库管理</a>
No 17. <a href="https://weibo.com/1402400261/O8sUBr3x2">MiraData：大规模视频数据集，具有长时长和结构化描述</a>
No 18. <a href="https://weibo.com/1402400261/O8tTe2lcP">BeeTrove：349K 个 OpenAI 自定义 GPTs 的数据</a>
No 19. <a href="https://weibo.com/1402400261/O89iJc55p">Search4All：开源版Perplexity，基于 LLM 和搜索引擎构建的平台，具有可定制且美观的界面，支持共享缓存搜索结果</a>
No 20. <a href="https://weibo.com/1402400261/O887qiGgi">LLM入门：不涉及太多数学和术语的LLM基本原理通俗讲解</a>
No 21. <a href="https://weibo.com/1402400261/O8kdOl9zC">几篇论文实现代码</a>
No 22. <a href="https://weibo.com/1402400261/O8p1d0cIM">[CL]《Training LLMs over Neurally Compressed Text》</a>
No 23. <a href="https://weibo.com/1402400261/O88YTFxtQ">Portr：一种隧道解决方案，可将本地 HTTP 和 TCP 连接公开到互联网。它利用 SSH 远程端口转发在底层进行安全隧道连接。Portr 主要面向希望在公共 URL 上暴露开发服务器的小型团队</a>
No 24. <a href="https://weibo.com/1402400261/O8pmigBsm">[RO] Embodied Neuromorphic Artificial Intelligence for Robotics: ......</a>
No 25. <a href="https://weibo.com/1402400261/O88jYlBat">只需250美元的低成本开源机械臂</a>
No 26. <a href="https://weibo.com/1402400261/O8p356KHE">[CL]《ReFT: Representation Finetuning for Language Models》</a>
No 27. <a href="https://weibo.com/1402400261/O8pvuhX15">[CL] The Probabilities Also Matter: A More Faithful Metric for ......</a>
No 28. <a href="https://weibo.com/1402400261/O8prC2QRE">[CL] Learning to Plan and Generate Text with Citations 网页链接 通过引入问题序列作为生成蓝图，探索了规划型模型的归因能力，发现相比标准序列到序列，规划明显提升了长篇问答的输出质量和引文准确率。</a>
No 29. <a href="https://weibo.com/1402400261/O86gyEnxx">[LG]《Prompt-prompted Mixture of Experts for Efficient LLM Generation》</a>
No 30. <a href="https://weibo.com/1402400261/O7H1vwTXM">'因果推断：从概念到实践 - Causal Inference for the Brave and True的中文翻译版。全部代码基于Python，适用于计量经济学、量化社会学、策略评估等领域。英文版原作者：Matheus ......</a>
No 31. <a href="https://weibo.com/1402400261/O8sZuiLxz">EasyContext：旨在提供一种方法，使用现有的技术来实现长上下文的语言模型，同时最大限度地减少所需的硬件</a>
No 32. <a href="https://weibo.com/1402400261/O8tQxEkO3">Multi-agent Quadruped Environment：多Agent四足机器人环境，支持定位控制或仅高层规划学习</a>
No 33. <a href="https://weibo.com/1402400261/O8p2xdlEa">[LG]《Rethinking Teacher-Student Curriculum Learning through the Cooperative Mechanics of Experience》</a>
No 34. <a href="https://weibo.com/1402400261/O8ufU9kYU">Meta率先扩大AI内容标记范围</a>
No 35. <a href="https://weibo.com/1402400261/O897H8gYD">pyspi: 提供了简单到复杂的方法来评估两个时间序列之间关系的Python库</a>
No 36. <a href="https://weibo.com/1402400261/O8p0r6RNO">早！[太阳] ​</a>
No 37. <a href="https://weibo.com/1402400261/O890w8HGO">FreeGPT35：基于免费且无限次调用的 GPT-3.5-Turbo API 服务构建的登录界面自由聊天 Web 应用程序</a>
No 38. <a href="https://weibo.com/1402400261/O8g35EtUJ">[LG] Domain Generalization through Meta-Learning: A Survey ......</a>
No 39. <a href="https://weibo.com/1402400261/O8tSg6xbJ">SCIN Dataset：SCIN 数据集包含来自美国互联网用户的 10,000+ 张皮肤疾病图像</a>
No 40. <a href="https://weibo.com/1402400261/O8h1oCNt9">128k上下文+多语言+工具：Cohere开放企业级应用大模型Command R+</a>
No 41. <a href="https://weibo.com/1402400261/O8iqA9D3s">JetMoE: 基于开源数据集训练的大型语言模型，可以与 LLaMA2-7B 等高性能模型相媲美，仅需较少的训练资金(0.1M Dollars)</a>
No 42. <a href="https://weibo.com/1402400261/O7OnL0nLA">用AutoTrain微调Mixtral 8x7B实战</a>
No 43. <a href="https://weibo.com/1402400261/O7K8fh9af">'stable-diffusion-tutorial - 全网最全Stable Diffusion全套教程，从入门到进阶' ......</a>
No 44. <a href="https://weibo.com/1402400261/O8p392NYd">提出表示微调框架，发现相比调整权重，直接对语言模型的表示进行有针对性的干预可以以更少的参数实现类似或更好的下游任务表现。//[CL]《ReFT: Representation Finetuning for Language Models》</a>
No 45. <a href="https://weibo.com/1402400261/O8todkCRB">Kyoo：基于网页的媒体库解决方案，旨在提供一个简单但功能强大的平台，用于组织和管理数字媒体内容</a>
No 46. <a href="https://weibo.com/1402400261/O8iQd2qwJ">SableDb：基于 RocksDb 存储引擎的 Key-Value NoSQL 数据库，与 Redis 协议兼容，旨在降低内存成本和增加比 Redis 更大的容量</a>
No 47. <a href="https://weibo.com/1402400261/O8pCZ8LXr">#免费##抽奖# 携手@博文视点Broadview 送出3本《LangChain实战》</a>
No 48. <a href="https://weibo.com/1402400261/O8pzl1jvt">[CL] Blessing or curse? A survey on the Impact of Generative AI ......</a>
No 49. <a href="https://weibo.com/1402400261/O8p1Yj0pc">[CL]《An Incomplete Loop: Deductive, Inductive, and Abductive Learning in Large Language Models》</a>
No 50. <a href="https://weibo.com/1402400261/O7OqmCa13">“Datawhale - 专注于AI领域的开源组织，致力于构建一个纯粹的学习圈子，帮助学习者更好地成长” ​</a>
